scripts/dockerfiles/run.sh

```diff
@@ -78,9 +78,17 @@ service rsyslog start
 cron
 
 # Run custom post config role scripts
+# First run all the scripts called pre-start
+for fname in $(find /var/vcap/jobs/*/bin -name pre-start) ; do
```

Comment:
Shouldn't we at least `{{ range $job := range .role.Jobs }}`? Just wildcarding it there seems like a potentially bad idea.